charts/stable/homebridge/values.yaml

```diff
@@ -1,165 +1,72 @@
-# Default values for homebridge.
-
-replicaCount: 1
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
 
 image:
-  # -- Image repository
   repository: oznu/homebridge
-  # -- Image pull policy
+  tag: 3.3.0
   pullPolicy: IfNotPresent
-  # -- Overrides the image tag whose default is the chart appVersion.
-  # Possible values listed [here](https://hub.docker.com/r/oznu/homebridge/tags)
-  tag: "3.1.0"
-
-# -- Secrets to use when pulling the image
-imagePullSecrets: []
-
-nameOverride: ""
-fullnameOverride: ""
-
-# -- Specifies the strategy used to replace old Pods by new ones
-strategyType: Recreate
-
-# -- Specify the container timezone
-timezone: "UTC"
-# -- process userID the instance should run as
-puid: 1000
-# -- process groupID the instance should run as
-pgid: 1000
-
-config:
-  # -- Enable the Homebridge UI plugin
-  enableUI: true
-  # -- Additional Homebridge plugins to install at container startup such as `homebridge-hue`
-  plugins: []
-  # -- Additional Alpine packages to install at container statup such as `bash`
-  additionalPackages: []
 
-# -- Extra ENV vars to pass to the homebridge container
-extraEnvs: []
-
-serviceAccount:
-  # Specifies whether a service account should be created
-  create: true
-  # Annotations to add to the service account
-  annotations: {}
-  # The name of the service account to use.
-  # If not set and create is true, a name is generated using the fullname template
-  name: ""
-
-# -- Key-value pairs to add as pod annotations
-podAnnotations: {}
-
-podSecurityContext: {}
-  # fsGroup: 2000
-
-securityContext: {}
-  # capabilities:
-  #   drop:
-  #   - ALL
-  # readOnlyRootFilesystem: true
-  # runAsNonRoot: true
-  # runAsUser: 1000
+# Possible values listed [here](https://hub.docker.com/r/oznu/homebridge/tags)
```

Comment:
```suggestion
# Possible values listed here: https://github.com/oznu/docker-homebridge#parameters
```

That link points to the available tags. I think this is the correct link for the env vars. I removed the markdown formatting, as I don't think it's being rendered by helmdocs.